adding written assessment tips to JOA section (#294)
usds/SME-QA-Website: pages/phases/job-analysis/creating-assessments-from-competencies.md

See a longer list at [this reference link](https://www.opm.gov/policy-data-oversight/assessment-and-selection/other-assessment-methods/).

## Written Assessments Tips

> - The assessment type should be appropriate to evaluate the applicant's proficiency level within the examined competencies.

Reviewer: ```suggestion
- The assessment type should be appropriate to evaluate the applicant's proficiency level within the pre-defined competencies.
```

- Ideally, you will have time during the workshop for live testing of the assessment questions. If you end up revising the assessment after the workshop, the revised assessment will need to be tested.

> - You should as part of testing record the time it will take to conduct the written assessment and added that to the job announcement.

Reviewer: ```suggestion
- You should as part of testing record the time it will take to conduct the written assessment and added that to the job announcement and to the assessment instructions.
```

> - You should provide an estimated timeframe in which to expect a written assessment will be sent out on the JOA (e.g., "If you are selected to move forward, you should expect to receive a written assessment within approximately 3-5 days of this announcement closing."

Reviewer: ```suggestion
- You should provide an estimated timeframe within the JOA in which applicants can expect a written assessment to be sent (e.g., "If you are selected to move forward, you should expect to receive a written assessment within approximately 3-5 days of this announcement closing."
```
